tests/testplotinstantiation.py

```diff
@@ -356,7 +356,7 @@ def setUp(self):
     def tearDown(self):
         Store.current_backend = self.previous_backend
         Callback._comm_type = comms.JupyterCommJS
-        mpl_renderer.comms['default'] = self.default_comm
+        bokeh_renderer.comms['default'] = self.default_comm
         Callback._callbacks = {}
```

Comment:
Bit surprised by this. I would have thought either you want to set it for both mpl and bokeh...or alternatively it was only ever meant for bokeh and was always wrong?

Reply:
It was wrong before. I just came across it and snuck the fix in here.

Reply:
I suppose the incorrect teardown didn't mess up the tests (which I suppose it could have) otherwise it would have been immediately spotted...